pymc3/distributions/dist_math.py

```diff
@@ -156,9 +157,9 @@ def log_normal(x, mean, **kwargs):
     elif rho is not None:
         std = rho2sd(rho)
     else:
-        std = tau**(-1)
+        std = tau**(f(-1))
     std += eps
-    return c - tt.log(tt.abs_(std)) - (x - mean) ** 2 / (2 * std ** 2)
+    return f(c) - tt.log(tt.abs_(std)) - (x - mean) ** f(2) / (f(2) * std ** f(2))
```

Comment:
Is that really necessary? I think python floats are not the problem, only numpy.